nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -223,7 +245,7 @@ def input_filter_func(point):
             node_name, input_filter_func, self._algorithm_key
         ):
             statistics = tensor_collector.get_statistics()
-            input_fp.extend(statistics.mean_values)
+            input_fp.extend(Tensor(statistics.mean_values))
```

Comment:
Yes, but i dont want to refactor all in one PR. In this PR fixed ptq for cuda and shows how to use Tensor on real code.

Reply:
IMO, we should provide the changes in case of UX improvements or developing/code support improvements.
Providing changes only `for demonstration` is not good.
So, this is why I still can't understand why should we add a new wrapper for all tensors in this PR.